src/tests/ReactElementAdapter.spec.jsx

```diff
@@ -278,7 +278,8 @@ describe('ReactElementAdapter', () => {
         it('returns the expect.it function for an expect.it content assertion', () => {
 
             const component = <span>{expect.it('to be a string')}</span>;
-            expect(adapter.getChildren(component), 'to equal', [ expect.it('to be a function')] )
```

Comment:
Yeah, I was just looking at that too, I think we can drop the peerDep of react entirely from that - I think it might have been needed originally, but isn't anymore